src/components/molecules/Common/AssetModal/AssetCard/index.tsx

```diff
@@ -57,16 +59,17 @@ const Wrapper = styled(Flex)<{ selected?: boolean; cardSize?: CardSize }>`
   box-shadow: 0 6px 6px -6px ${props => props.theme.colors.other.black};
   border: 1px solid
     ${props => (props.selected ? `${props.theme.assetCard.highlight}` : "transparent")};
-  margin: ${({ cardSize }) => (cardSize === "small" ? "5px" : "10px")};
+  margin: 12px;
   padding: ${({ cardSize }) =>
     cardSize === "small" ? "8px" : cardSize === "medium" ? "12px" : "20px"};
   width: ${({ cardSize }) =>
-    cardSize === "small" ? "104px" : cardSize === "medium" ? "163px" : "218px"};
+    cardSize === "small" ? "104px" : cardSize === "medium" ? "192px" : "274px"};
   height: ${({ cardSize }) =>
-    cardSize === "small" ? "104px" : cardSize === "medium" ? "168px" : "234px"};
+    cardSize === "small" ? "104px" : cardSize === "medium" ? "186px" : "257px"};
```

Comment:
I am sure you have, but did you make sure the styles are still okay? I am pretty sure the values you added are the values in Figma, but the reason I initially reduced them was taking into consideration the padding + any discrepancies because of borders,etc that might be adding width or height unexpectedly(compared to the Figma design).

Reply:
yes, yes, don't worry. I made them box-sizing. and adjuest them